statsd/buffer_pool.go

```diff
@@ -0,0 +1,30 @@
+package statsd
+
+import (
+	"bytes"
+	"sync"
+)
+
+type bufferPool struct {
+	sync.Pool
```

Comment:
Curious why you didn't use a pointer to sync.Pool, as the Get/Put methods are pointer methods (dispatch still works of course though).